helm/cluster/templates/clusterapi/_helpers_files.tpl

```diff
@@ -61,10 +61,10 @@
 {{- end }}
 
 {{- define "cluster.internal.kubeadm.files.kubelet" }}
-- path: /etc/kubelet-configuration.yaml
-  permissions: "0700"
+- path: /etc/kubernetes/patches/kubeletconfiguration.yaml
```

Comment:
Does the name matter? Otherwise I'd keep the "-" in between.

Reply:
Yes, it does. There a schema backing it:

https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/#patches